### PR TITLE
Add links to the worker explorer at appropriate places in the docs

### PR DESCRIPTION
**Pull request by djmitche**

### CODE REVIEW

File: src/manual/task-execution/workers.md [hunk 1]
```diff
@@ -18,6 +18,9 @@ Each individual worker is identified by a `<workerGroup>/<workerId>` pair,
 identified in the task run information.  This information can be useful to
 identify misbehaving workers by examining the runs of failed tasks.
 
+You can see a list of currently-running workers, arranged by worker type, at
+https://tools.taskcluster.net/provisioners.
```
Comment (helfi92) [inline src/manual/task-execution/workers.md]: The list of workers arranged by worker-type is in `https://tools.taskcluster.net/provisioners/:provisionerId/worker-types/:workerType`

File: src/manual/tasks/workertypes.md [hunk 2]
```diff
@@ -19,4 +19,7 @@ interchangeable and have identical configurations.
 The format of a task's `payload` property is specific to the worker that will
 execute it, so defining a task requires knowledge of worker type's
 configuration. If given a task with an inappropriate payload, a worker will
-resolve the task with the reason `malformed-payload`.
\ No newline at end of file
+resolve the task with the reason `malformed-payload`.
+
+You can explore the available worker types (organized by `provisionerId`) at
+https://tools.taskcluster.net/provisioners.
```
Comment (helfi92) [inline src/manual/tasks/workertypes.md]: `https://tools.taskcluster.net/provisioners/:provisionerId/worker-types`

Comment (djmitche) [reply src/manual/tasks/workertypes.md]: I want to make links, though, so the user can click and find what they are looking for without editing the URLs.  Maybe I can phrase that differently..

**Comment by djmitche** [comment]
How's that?

**Pull request merged**